src/server/utils/oneAppDevCdn.js

```diff
@@ -0,0 +1,208 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
side note: can this be something like https://github.com/americanexpress/one-app/blob/v6.7.1/src/server/devHolocronCDN.js#L17 instead?
```suggestion
// file only used during development
/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true }] */
```